### DISCUSSION TITLE
documentation bug

### DISCUSSION THREAD

**Discussion by jettero** [goreleaser/goreleaser#2926] [Q&A] [answered]
I asked about this as a comment on some other issue, but I can't now find that comment or the issue. I believe it was a closed issue, whatever it was.

I want to cross compile a binary using CGO. The dependency that requires this has a library (.a) for linux and another for darwin and it compiles fine if I do everything by hand. I believe features were added to goreleaser to allow this to work, but even after several hours I just can't get it to work.

It seems someone was working on an example, but never finished it. I wonder if it's a known work in progress or if it got forgotten, so I'm raising this issue as a reminder to publish this example some day (i.e., please help me, I can't figure this out).

https://github.com/goreleaser/goreleaser-cross-example

https://goreleaser.com/cookbooks/cgo-and-crosscompiling/

I love goreleaser and think there's great work here. Sorry to complain about a missing/unfinished doc item.

**Comment by caarlos0** [comment, marked as answer]
You might want to look into https://github.com/goreleaser/goreleaser-cross 

Your issue does not explain what are the configs you tried, so its very hard to help further...

cc/ @troian

**Reply by jettero** [reply]
Well, the problem I'm trying to solve is somewhat irrelevant. I was mainly commenting on the fact that the docs on the limitations of CGO are very brief and just say it's an unclean goreleaser config, then link to a literally empty example.

I'm specifically not looking for help with my config file, I'm asking for the examples that are linked from the docs.

**Reply by jettero** [reply]
On further reflection, if those docs don't exist and are not going to ever exist, probably the document should be changed to reflect that fact. Possibly they should be changed to direct people to goreleaser-cross instead.

I still think it's a ticket worthy doc bug because either the docs are flat wrong and should direct people to this other project repo, or they're wrong because they link to an empty example repo.

**Reply by troian** [reply]
I believe it has been solved. `golang-cross` became part of the `goreleaser` team recently thus there might be some disruptions with links.

**Reply by caarlos0** [reply]
yes, thanks @troian 

@jettero please let us know if that helps

**Reply by jettero** [reply]
OIC. Doc still in flux. I thought it might be that, but I wondered if it was simply forgotten.

I'm still new to Go, but I expect it's a bit niche to build .a files in with gorelesser

**Reply by caarlos0** [reply]
marking as answer as it was fixed 

thanks, y'all
